SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1747,7 +1748,7 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 
 	if (SpatialGDKSettings->bRunSpatialWorkerConnectionOnGameThread)
 	{
-		if (Connection != nullptr && Connection->IsConnected())
+		if (Connection != nullptr)
```

Comment:
My only concern with doing this everywhere is that it's less explicit. And generally when seeing !=nullptr I'm not entirely sure why something might be nullptr. It could be that using ConnectionManager->IsConnected() would be better but then you want to be querying the workerconnection unless you are the game instance.

Reply:
Really except for startup, this should always be valid.   (There's an open reconnect discussion, but I think that's outside of the scope.)  Would you feel better if this method (and one other in the tick processing) just had an early out for no connection and then we assume that it's valid any other time?

Reply:
I think leaving the check in is fine for now. It was more on the != nullptr, vs ->IsConnected. But If this is going to be a pain I think this is fine.